### PR TITLE
Gough Book 7, 1-2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: history-of-the-quakers-v4/original/07-book-7-chapter-1.adoc [hunk 1]
```diff
@@ -51,21 +60,31 @@ And as it hath been our known principle to live peaceably under government,
 so we hope it will always be our practice, through God`'s assistance,
 to approve ourselves, with hearty affection, thy faithful and dutiful subjects.
 
+[.small-break]
+'''
+
 May the wonderful counsellor and great preserver
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/07-book-7-chapter-1.adoc:66]: This paragraph was indented more than the rest in the original and I'm not sure why. It seems to be okay aligned with the rest of the embed?

File: history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc [hunk 2]
```diff
@@ -386,6 +395,10 @@ the lords were to go into a grand committee on the bill,
 but were prevented by an unexpected petition from some of the London clergy,
 which was presented by the Archbishop of York, and was as follows:
 
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 To the right honourable the Lords Spiritual and Temporal in Parliament assembled.
 
 The humble petition^
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc:404]: So marking this as anything ( [.blurb], [.old-style], etc.) causes the footnote to be in the text in the PDF so I wasn't sure what to do.

File: history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc [hunk 3]
```diff
@@ -1284,17 +1317,19 @@ and applications for their relief having been ineffectually
 made to the government of that province,
 gave occasion to the following petition to the government at home.
 
-A petition to the King in the cause of some friends under sufferings in New England.
+[.embedded-content-document.letter]
```
Comment (maijapapaya) [inline history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc:1320]: So the top of this whole thing might need a little work.

File: history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc [hunk 4]
```diff
@@ -470,8 +484,11 @@ themselves to your lordships great wisdom,
 may induce your lordships not to give your consent
 to the passing of this bill into a law,
 
+[.signed-section-context-close]
```
Comment (jaredh159) [inline history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc:487]: I think this should be a `[.signed-section-closing]` instead.

File: history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc (hunk 2 above)
Comment (jaredh159) [reply history-of-the-quakers-v4/original/08-book-7-chapter-2.adoc:404]: Yeah we have a problem with footnotes on headings.  Why don't you add `[.centered.emphasized]` above line 404 for now.  That should get it looking pretty good, and I made a [note to fix it](https://github.com/friends-library/friends-library/issues/85#issuecomment-636021593) whenever I come up with a way to handle footnotes in headings.

**Comment by jaredh159** [comment]
Note to self: i want to work on the one "present" section a bit after merge.

**Comment by jaredh159** [comment]
looks good, Maija, merging. 👍

**Pull request merged**